app/src/main/java/de/danoeh/antennapod/activity/AudioplayerActivity.java

```diff
@@ -569,6 +569,8 @@ protected void onPlaybackSpeedChange() {
 
     private void updateButPlaybackSpeed() {
         if (controller != null && controller.canSetPlaybackSpeed()) {
+            //Feed feed = <must somehow get feed here!!>
+            //butPlaybackSpeed.setText(feed.getPreferences().getCurrentPlaybackSpeed());
```

Comment:
OK, I need to review a little bit all the media player classes and how they are called. My knowledge is a bit lacking there.
Due to other commitments, It will take me at least a few days, if not a week. I'll keep rebasing as needed.